pkg/api/const.go

```diff
@@ -61,8 +61,10 @@ const (
 	Kubernetes162 OrchestratorVersion = "1.6.2"
 	// Kubernetes166 is the string constant for Kubernetes 1.6.6
 	Kubernetes166 OrchestratorVersion = "1.6.6"
+	// Kubernetes166 is the string constant for Kubernetes 1.6.6
+	Kubernetes170 OrchestratorVersion = "1.7.0"
 	// KubernetesLatest is the string constant for latest Kubernetes version
-	KubernetesLatest OrchestratorVersion = Kubernetes166
+	KubernetesLatest OrchestratorVersion = Kubernetes170
```

Comment:
while this is true it is the latest version we might want to mark 1.6.6 for a while as default until we have a little bake time on 1.7.0